### PR TITLE
Rewrite case tile pull-down test to use `assertXmlPartialEqual`

### PR DESCRIPTION
**Pull request by NoahCarnahan**
Follow on from https://github.com/dimagi/commcare-hq/pull/5832#issuecomment-78218425.

### CODE REVIEW

File: corehq/apps/app_manager/tests/test_suite.py [hunk 1]
```diff
@@ -409,8 +411,62 @@ def test_case_detail_tabs(self):
...
+                format='date',
+                case_tile_field='date'
+            ),
+        ]
+
+        form = app.new_form(0, "Untitled Form", None)
+        form.xmlns = 'http://id_m0-f0'
+        form.requires = 'case'
+
+        self.assertXmlPartialEqual(self.get_xml('case_tile_pulldown_session'), app.create_suite(), "./entry/session")
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_suite.py]: E501 line too long (117 > 115 characters)

**Comment by snopoke** [comment]
Nice, I think you should also add the `detail-inline` field to the `SessionDatum` sort ordering: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/suite_xml.py#L237-237

File: corehq/apps/app_manager/suite_xml.py [hunk 2]
```diff
@@ -236,7 +236,7 @@ class Instance(IdNode, OrderedXmlObject):
 
 class SessionDatum(IdNode, OrderedXmlObject):
     ROOT_NAME = 'datum'
-    ORDER = ('id', 'nodeset', 'value', 'function', 'detail_select', 'detail_confirm', 'detail_persistent')
+    ORDER = ('id', 'nodeset', 'value', 'function', 'detail_select', 'detail_confirm', 'detail_persistent', 'detail_inline')
```
Comment (dimagimon) [inline corehq/apps/app_manager/suite_xml.py]: E501 line too long (123 > 115 characters)

**Comment by NoahCarnahan** [comment]
@snopoke Good point, added.

**Pull request merged**